expression/builtin_cast.go

```diff
@@ -2007,6 +2007,17 @@ func minimalDecimalLenForHoldingInteger(tp byte) int {
 	}
 }
 
+// typeString returns true if precision cannot be determined
+func typeString(fieldType byte) bool {
```

Comment:
Can we just use `types.IsString`?

Reply:
TypeUnspecified is not the expected type，which satisfies  types.IsString